### PR TITLE
myla311

### PR DESCRIPTION
**Pull request by leicao-me**
inital commit

### CODE REVIEW

**Comment by leicao-me** [comment]
second job done.

File: dags/myla311/dag_311_outlier_detector.py [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+sql_pull_data = \
+    """
+    COPY myla311_main TO '{}' DELIMITER ',' CSV HEADER;
+    """.format(os.path.abspath(filename))
+
+# airflow DAG arguments
+args = {'owner': 'airflow',
+    'start_date': airflow.utils.dates.days_ago(7),
+    'provide_context': True,
+    # 'retries': 2, # will be set after testing
```
[the remark this answers is not in the document]
Comment (leicao-me) [reply dags/myla311/dag_311_outlier_detector.py]: also add ITADATA@lacity.org

File: dags/myla311/dag_311_outlier_detector.py [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+        body = MIMEText(message, 'plain')  # convert the body to a MIME compatible string
+        msg.attach(body)
+
+        server = smtplib.SMTP(smtpserver)
+        server.starttls()
+        server.login(login, password)
+        server.sendmail(msg['From'], [to_addr_list, cc_addr_list], msg.as_string())
+        return "Outliers founded. Alert Email sent. Success!"
+
+    return "No alert generated. Email not sent."
```
[the remark this answers is not in the document]
Comment (leicao-me) [reply dags/myla311/dag_311_outlier_detector.py]: okay to use `from airflow.utils.email import send_email`

File: dags/myla311/retrieve_update_myla311.py [hunk 3]
```diff
@@ -12,7 +12,8 @@
 import boto3
 import botocore
 import os
-from config import MY_APP_TOKEN, USERNAME, PASSWORD, CONNECTION_ID
```
[the remark this answers is not in the document]
Comment (leicao-me) [reply dags/myla311/retrieve_update_myla311.py]: Do you mean store them in airflow.cfg?

[the remark this answers is not in the document]
Comment (hunterowens) [reply dags/myla311/retrieve_update_myla311.py]: store using Airflow Connections, set inside the UI. 

[example](https://www.astronomer.io/guides/airflow-ui/)

**Pull request merged**